src/vec2.rs

```diff
@@ -72,6 +80,23 @@ impl Vec2 {
     ///
     /// This is similar to `self.hypot2().sqrt()` but defers to the platform `hypot` method, which
```

Comment:
Can we make this be a link to `f64::hypot`? I for one didn't know this method existed previously